index.html

```diff
@@ -1158,6 +1176,23 @@ <h3>Role Mapping Table</h3>
 						<span class="property">AXSubrole: <code>AXLandmarkMain</code></span><br />
 					</td>
 				</tr>
+				<tr id="role-map-mark">
+					<th><a class="role-reference" href="#mark"><code>mark</code></a></th>
+					<td class="role-msaa-ia2">
+						<span class="property">Role: <code>ROLE_SYSTEM_GROUPING</code></span><br />
+						<span class="property">Role: <code>IA2_ROLE_MARK</code></span><br />
+						<span class="property">Object Attribute: <code>xml-roles:mark</code></span>
+					</td>
+					<td class="role-uia">
+						<span class="property">Control Type: <code>Group</code></span>
+					</td>
+					<td class="role-atk">
+						<span class="property">Role: <code>ROLE_MARK</code></span>
```

Comment:
1. The xml-roles:mark should also be exposed for ATK. We both know it will whether it says so in this spec or not, but let's add it. It will be needed because there isn't (yet) a ROLE_MARK in ATK. And even when it's there, we cannot count on all users having the latest version. Plus, you did the xml-roles thing for comment and suggestion for ATK....

2. I've got a merge request to add this and suggestion to ATK: https://gitlab.gnome.org/GNOME/atk/merge_requests/36. I anticipate it landing soon. If for some reason it doesn't, we don't want to land your change in Core-AAM as-is. We'd either block it on ATK or remove the not-yet-existing ATK role from your change here.

Reply:
We can block this on the ATK change. I just want to have this ready to go in. I'll add xml-roles.

Reply:
FWIW, I committed the ATK changes. I've also got merge requests for AT-SPI2 (core, bridge, and python bindings). So with my platform-API-owner/maintainer hat on, I think we're set for Linux.

Thanks again for doing this PR!